src/HeaderBar.scss

```diff
@@ -0,0 +1,5 @@
+@import "@material/top-app-bar/mdc-top-app-bar";
```

Comment:
Is there any way we can use the precompiled CSS files from the npm module's `dist/` folder, so we don't need to recompile the Sass for every component?